src/drive/web/modules/navigation/duck/reducer.js

```diff
@@ -360,18 +360,32 @@ export const getFileById = ({ view }, id) => {
 
 const isRootFolder = folder => folder.id === ROOT_DIR_ID
 
-const ensureFileHavePath = ({ view }, file) => ({
+export const ensureFileHavePath = ({ view }, file) => ({
   ...file,
+  displayedPath: getDisplayedFilePath({ view }, file),
   path: getFilePath({ view }, file)
 })
 
+/*
+  TODO: deprecate and remove getFilePath
+*/
 export const getFilePath = ({ view }, file) => {
   const { displayedFolder } = view
```

Comment:
What do you think about adding a console.warn here with the trace of the call and a link to this discussion? Like that, you'll see where and when this function is called?

Reply:
I think this function is called by redux `connect`, so it will be called all the time :/